<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>598</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="LAYOUT_MAIN_LEFT" stretch="0,10">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="styleButton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Изменить стиль</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="LAYOUT_LIST_CHAT">
        <item>
         <widget class="QScrollArea" name="scrollAreaListChat">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContentsListChat">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>211</width>
             <height>523</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="LAYOUT_MAIN_RIGHT" stretch="10,1">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="LAYOUT_CHAT" stretch="0,1,10">
        <item>
         <widget class="QLabel" name="notifi">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="chatName">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="gridSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollAreaMessage">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>682</width>
             <height>405</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="LAYOUT_MESSAGE_INPUT">
        <item>
         <widget class="QTextEdit" name="message_panel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="messButtonPush">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="inputMethodHints">
           <set>Qt::InputMethodHint::ImhNone</set>
          </property>
          <property name="text">
           <string>Отправить</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
